### PR TITLE
Expanded for object based data [{id: x, text: 'label'}]

### PR DESCRIPTION
**Pull request by brookslyrette**

### CODE REVIEW

File: examples/src/components/Tags.js [hunk 1]
```diff
@@ -17,6 +18,20 @@ export default class Tags extends Component {
           options={{
             placeholder: 'search by tags',
           }} />
+          <br/>
+          <br/>
+          Using {'{'}id: 'x', text:'foo'{'}'}
+          <Select2
+            multiple={false}
+            data={[{text: 'Bug', id: 0}, {text: 'Feature', id: 1}, {text: 'Documents', id: 2}, {text: 'Discussion', id: 'UUID'}]}
```
Comment (rkit) [inline examples/src/components/Tags.js]: Please check the [js code style](https://github.com/rkit/react-select2-wrapper/blob/master/.eslintrc): Max length of line should be max 120

File: src/components/Select2.js [hunk 2]
```diff
@@ -50,7 +50,12 @@ export default class Select2 extends Component {
     return (
       <select multiple={this.props.multiple}>
         {this.props.data.map((item, k) => {
-          return (<option key={'option-' + k}>{item}</option>);
+          if(typeof item === 'string' || ((!!item && typeof item === 'object') && Object.prototype.toString.call(item) === '[object String]')) {
```
Comment (rkit) [inline src/components/Select2.js]: Please check the [js code style](https://github.com/rkit/react-select2-wrapper/blob/master/.eslintrc): http://jmp.sh/UUbq1Ub (only errors)

**Comment by rkit** [comment]
Thanks! I added a few comments…

File: README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
-# react-select2-wrapper
+# react-select2-data-wrapper
```
Comment (rkit) [inline README.md]: Why change the name?

**Comment by brookslyrette** [comment]
Sorry I committed more than I mean too. I'll clean up the code style and revert the name changes.

I just did that to get it on npmjs for a project I'm actively working on this morning.

**Comment by brookslyrette** [comment]
@rkit This should fix the rename and eslint issues.

**Comment by rkit** [comment]
@brookslyrette Thanks!

**Pull request merged**